src/main/java/davidsar/gent/stubjars/Main.java

```diff
@@ -84,10 +89,14 @@ private static void createDirectoryTree(StubJars build) {
         log.info("stub_src directory tree creation finished");
     }
 
-    private static void createSourceFiles(StubJars build) {
+    private static boolean createSourceFiles(StubJars build) {
         log.info("Starting creation of stub_src files");
-        build.createSourceFiles();
+        boolean sourceFilesCreated = build.createSourceFiles();
         log.info("Creation of stub_src files finished");
+        if (!sourceFilesCreated) {
+            log.error("Source file creation was interrupted due to an error");
+        }
```

Comment:
This exact error message also gets printed by this method's caller under the exact same circumstances, so printing it here should probably get removed.